guides/doc-Provisioning_Guide/topics/Virt-VMWare.adoc

```diff
@@ -476,15 +476,16 @@ You can use the xref:Provisioning_Virtual_Machines_in_VMware_vSphere-Adding_VMwa
 If you deploy {Project} with the {SmartProxy} templates feature, you must configure {Project} to recognize hosts' IP addresses forwarded over the X-Forwarded-For HTTP header to serve correct template payload.
 
 For security reasons, {Project} recognizes this HTTP header only from localhost.
-For each individual {SmartProxy}, you must configure a regular expression to recognize hosts' IP addresses.
-From the {ProjectWebUI}, you can do this by navigating to *Administer* > *Settings* > *Provisioning*, and changing the *Remote address* setting.
-From the CLI, you can do this by entering the following command:
+For each individual {SmartProxy}, you must configure a setting to recognize hosts' IP addresses.
+To add an IP address (e.g. 192.0.2.10), use the the following installer command:
 
 [options="nowrap" subs="+quotes,attributes"]
 ----
-# hammer settings set --name remote_addr --value '(localhost(4|6|4to6)?|192.168.122.(1|2|3))'
+# {foreman-installer} --foreman-trusted-proxies 127.0.0.1/8 --foreman-trusted-proxies ::1 --foreman-trusted-proxies 192.0.2.10
 ----
 
+The localhost entries are required for Apache reverse HTTP proxy, do not omit them.
```

Comment:
I wonder if "Apache reverse HTTP Proxy" could be simplified to "Apache". Or even leave it out altogether:
```suggestion
The localhost entries are required, do not omit them.
```